x/ibc/ibc_test.go

```diff
@@ -31,6 +33,8 @@ const (
 	channelOrder   = channeltypes.ORDERED
 	channelVersion = "1.0"
 
+	height = 10
```

Comment:
nit, can we use the ibc testing package here? 🙏

Reply:
this is just used by `genesis_test.go`, fine for now. `handler_test.go` uses ibc testing pkg